assets/prometheus-k8s/rules.yaml

```diff
@@ -274,6 +274,8 @@ spec:
     - expr: sum(rate(container_cpu_usage_seconds_total{container="",pod!=""}[5m]))
         BY (pod, namespace)
       record: pod:container_cpu_usage:sum
+    - expr: sum(kube_pod_container_status_last_terminated_reason{reason="OOMKilled",job="kube-state-metrics"})
```

Comment:
To avoid double accounting in corner cases:

`sum(max by(namespace, pod, container) (kube_pod_container_status_last_terminated_reason{...}))`